src/pylon_camera/pylon_camera_node.cpp

```diff
@@ -34,6 +34,7 @@
 #include <vector>
 #include "boost/multi_array.hpp"
 
+
```

Comment:
Try not to make whitespace changes in a feature PR unless there is a strong reason to. They make it harder to find the real changes in the diff.